addons/godot-traits/godot-traits.gd

```diff
@@ -35,6 +42,9 @@ func _unhandled_key_input(event: InputEvent) -> void:
     if Engine.is_editor_hint():
         if GTraitsEditorSettings.get_instance().get_gtraits_helper_regeneration_shortcut().matches_event(event):
             if event.is_released():
-                GTraitsFileSystem.get_instance().force_full_scan()
-                GTraitsHelperGenerator.get_instance().clear_and_regenerate()
-                _logger.info(func(): return "🎭 Godot Traits: GTraits script regenerated in '%s'" % GTraitsEditorSettings.get_instance().get_gtraits_helper_output_path())
+                regenerate_gtraits_script()
+
+func regenerate_gtraits_script():
```

Comment:
Since this is a private function, it should start with an `_`. I also like strong typing 😄 

```gdscript
func _regenerate_gtraits_script() -> void:
    pass
```